js/__internal/grids/grid_core/data_controller/m_data_controller.ts

```diff
@@ -1365,7 +1366,7 @@ export class DataController extends ControllerWithDataMixin {
           isCustomLoading: true,
           storeLoadOptions: { isLoadingAll: true },
           loadOptions: {
-            filter: that.getCombinedFilter(),
+            filter: skipFilter ? undefined : that.getCombinedFilter(),
```

Comment:
Minor comment.

Will it be better if we'll use null here?
Because undefined - it's not defined, and null is an empty reference (empty filter in our case)

Reply:
I don't see any difference. Filter is undefined -> filter is not defined -> no filter. I'll change